src/image_user_layer.ts

```diff
@@ -98,8 +98,10 @@ import {
   ShaderControls,
 } from "#/widget/shader_controls";
 import { Tab } from "#/widget/tab_view";
+import { trackableFiniteFloat } from "#/trackable_finite_float";
 
 const OPACITY_JSON_KEY = "opacity";
+const GAIN_JSON_KEY = "gain";
```

Comment:
maybe rename to volumeRenderingGain since it only applies to volume rendering?

Also when I've created this control manually as a UI control, I've typically found it convenient to give it an exponential scale so that it can more easily cover a larger range, i.e. the slider ranges from -10 to 10 and the actual gain is `exp(gain)`.  But that could also be handled purely in the slider control.  Maybe you've also found the linear gain to work okay.

Reply:
Great, thanks! Linear worked pretty well, but I prefer the exponential scale after trying it out, thanks for the suggestion on that.
Not sure if we should hint at the scale in the UI somehow now, or if the name Gain (3D) is clear enough. Happy either way, but let me know if you have thoughts on that